lib/app/common/app_page/app_page.dart

```diff
@@ -191,9 +198,16 @@ class _AppPageState extends State<AppPage> {
       Clipboard.setData(ClipboardData(text: appData.website));
     }
 
+    final onPublisherSearch =
+        widget.enableSearch == false || !widget.initialized
+            ? null
+            : () => Navigator.of(context)
+                .pop(searchByPublisher(widget.appData.publisherUsername));
```

Comment:
This adds a small delay, but otherwise WorksForMe™
```suggestion
    final onPublisherSearch =
        widget.enableSearch == false || !widget.initialized
            ? null
            : () async {
                await searchByPublisher(widget.appData.publisherUsername);
                if (context.mounted) {
                  Navigator.of(context).pop();
                }
              };
```